ci/conda/recipes/morpheus/morpheus_build.sh

```diff
@@ -45,6 +54,7 @@ CMAKE_ARGS="-DCMAKE_BUILD_RPATH_USE_ORIGIN=ON ${CMAKE_ARGS}"
 CMAKE_ARGS="-DCMAKE_CUDA_ARCHITECTURES=${CMAKE_CUDA_ARCHITECTURES=-"RAPIDS"} ${CMAKE_ARGS}"
 CMAKE_ARGS="-DPython_EXECUTABLE=${PYTHON} ${CMAKE_ARGS}"
 CMAKE_ARGS="-DPYTHON_EXECUTABLE=${PYTHON} ${CMAKE_ARGS}" # for pybind11
+CMAKE_ARGS="--log-level=VERBOSE ${CMAKE_ARGS}"
```

Comment:
Question: Was this left in intentionally?

Reply:
Yes, no reason not to have it on since its usually hidden and can really help if you are looking at the build logs.